src/app/frontend/logs/template.html

```diff
@@ -64,6 +64,7 @@
           'kd-logs-size-icon': !logService.getCompact()}">text_fields</mat-icon>
         </button>
         <button mat-icon-button
+                matTooltip="Show timestamp"
```

Comment:
Please change to "Show timestamps" as they are displayed in each line.

/approve

Reply:
Okay, I've changed it to "Show timestamps".